sql/src/test/java/org/apache/druid/sql/calcite/NegativeTest.java

```diff
@@ -36,23 +37,52 @@
 import static org.junit.Assert.assertThrows;
 
 /**
- * Can be used to mark tests which are not-yet supported in decoupled mode.
+ * Can be used to mark tests which are not-yet supported for some reason.
+ *
+ * In case a testcase marked with this annotation fails - it means that the
+ * testcase no longer fails with the annotated expectation.
+ *
+ * During usage; the annotation process have to be added to the testclass.
+ * Ensure that it's loaded as the most outer-rule by using order=0 - otherwise
+ * it may interfere with other rules:
+ * <code>
+ *   @Rule(order = 0)
+ *   public NegativeTestProcessor decoupledIgnoreProcessor = new NegativeTestProcessor();
+ *
+ *   @NegativeTest
+ *   @Test
+ *   public void testA() {
+ *   }
+ * </code>
  *
- * In case a testcase marked with this annotation fails - it may mean that the
- * testcase no longer needs that annotation.
  */
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ElementType.METHOD})
-public @interface DecoupledIgnore
+public @interface NegativeTest
```

Comment:
I also didn't like `NegativeTest`...but failed to come up with a better name...

I feel like probably the main difference is that in a negative test - that's the expected behaviour; but in this case that's not really the case - its more like "how it is now" :D 

* `IgnoredTest`
    * I didn't wanted to reuse the word `Ignore` as that already have a meaning in the context of junit - and because its actually being run its not really "ignored"
* `FailingTest`
   * I think it may also hold the same confusion as `Negative`
* `FutureImprovementIgnore`
   * a bit long - but it describes it
* `NotYetSupported`
* `Fixme`
     * I started liking it a bit as it communicates clearly that its not the intended behaviour and I think it might also cause people to be curious about the annotation

what do you think which would be the best?

Reply:
Both `FixMe` and `NotYetSupported` seem good.